tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php

```diff
@@ -0,0 +1,421 @@
+    {
+        return array_merge(
+            $this->provideFixWithBreakCases(),
+            $this->provideFixWithContinueCases(),
+            $this->provideFixWithContinueCases(),
+            $this->provideFixWithReturnCases(),
+            $this->provideFixWithThrowCases(),
+            $this->provideFixWithYieldCases()
+        );
+    }
+
+    /**
+     * @dataProvider provideFixWithBreakCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithBreak($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'break',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithBreakCases()
+    {
+        return [
+            [
+                '<?php break 9000; ?>',
+            ],
+            [
+                '<?php break 9000; ?>',
+                '<?php break  9000; ?>',
+            ],
+            [
+                '<?php break /* foo */9000; ?>',
+                '<?php break/* foo */9000; ?>',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithContinueCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithContinue($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'continue',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithContinueCases()
+    {
+        return [
+            [
+                '<?php continue 9000; ?>',
+            ],
+            [
+                '<?php continue 9000; ?>',
+                '<?php continue  9000; ?>',
+            ],
+            [
+                '<?php continue /* foo */9000; ?>',
+                '<?php continue/* foo */9000; ?>',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithEchoCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithEcho($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'echo',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithEchoCases()
+    {
+        return [
+            [
+                '<?php echo 9000; ?>',
+            ],
+            [
+                '<?php echo 9000; ?>',
+                '<?php echo  9000; ?>',
+            ],
+            [
+                '<?php echo /* foo */9000; ?>',
+                '<?php echo/* foo */9000; ?>',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithGotoCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithGoto($expected, $input = null)
```

Comment:
looks like we can just have a single `testFix` method. we don't need separated method for fixing `echo`, separated method for fixing `continue` and so...

Reply:
I took a look at `BlankLineBeforeStatementFixerTest`, so I figured it would make sense to test individual configuration options.

Should I merge into one?

Reply:
you can test individual configs even when having only one `testFix`.
`testFix($expected, $input = null, array $config = [])` (we even sometimes have `expected, input, config, file`)

Reply:
Adjusted!